### DISCUSSION TITLE
How to wire my Touch Display to ESP8266 generic?

### DISCUSSION THREAD

**Discussion by DSRocker** [HASwitchPlate/HASPone#166] [Q&A] [answered]
Hi,
I use ESP8266 12F on a breakboard. How to connect my 3,5" TFT LCD Touch Display (ILI9481) ([LINK](https://www.ebay.de/itm/174949498297?var=474095516573)) to use HASwitchPlate?
Thanks!

**Comment by aderusha** [comment, marked as answer]
This project won't work at all without a Nextion display, generic LCDs aren't going to be compatible.  You can get the full BOM here: https://github.com/HASwitchPlate/HASPone/wiki/Building-your-own-HASPone
